tests/backup/backup_kdmp_test.go

```diff
@@ -1400,3 +1401,158 @@ var _ = Describe("{ExcludeInvalidDirectoryFileBackup}", func() {
 
 	})
 })
+
+// This TC takes 30 generic backups, deletes intermittent backups and validates the restores
+var _ = Describe("{IssueGenericBackupsAndRestoreInterleavedCopies}", func() {
```

Comment:
Go ahead and update the image.

Reply:
@kshithijiyer-px, sorry- image? you mean the TC?

Reply:
Yes the testcase.